lib/scala/kg/src/main/scala/stores/kg/neo4j/Neo4jKgStoreTransactionWrapper.scala

```diff
@@ -355,4 +357,14 @@ class Neo4jKgStoreTransactionWrapper(configuration: Neo4jStoreConfiguration, tra
 
   private def toTransactionRunParameters(map: Map[String, Any]) =
     map.asJava.asInstanceOf[java.util.Map[String, Object]]
+
+  private def writeNodePageRanks =
+    transaction.run(
+      """
+        |CALL gds.pageRank.write({
+        |nodeProjection: 'Node',
```

Comment:
Are these projections minimal? This is creating an in-memory data structure, so minimizing the projection will be necessary for dealing with the full CSKG.

Reply:
I think they are, but not 100% sure